relay-monitors/src/lib.rs

```diff
@@ -150,35 +148,15 @@ pub fn process_check_in(payload: &[u8]) -> Result<Vec<u8>, ProcessCheckInError>
         check_in.status = CheckInStatus::Unknown;
     }
 
-    check_in.monitor_slug = slugify(&check_in.monitor_slug);
+    check_in.monitor_slug.truncate(SLUG_LENGTH);
```

Comment:
This will panic if `SLUG_LENGTH` is not at a character boundary: https://doc.rust-lang.org/std/string/struct.String.html#method.truncate.

Something like this should work:

```suggestion
    if let Some(overflow) = check_in.monitor_slug.char_indices().skip(SLUG_LENGTH).next()) {
        check_in.monitor_slug.truncate(overflow.0);
    }
```

Reply:
👍 good call